cypress/integration/tests/controls/businessservices/crud.test.ts

```diff
@@ -35,4 +41,41 @@ describe("Business service CRUD", () => {
         // Assert that Business service is deleted
         notExists(businessService.name);
     });
+
+    it("Business service CRUD with owner", function () {
+        // Create owner - stakeholder
+        const stakeholder = new Stakeholders(data.getEmail(), data.getFullName());
+        stakeholder.create();
+        cy.wait("@postStakeholder");
+
+        // Create new Business service with owner attached
+        const businessService = new BusinessServices(
+            data.getCompanyName(),
+            data.getDescription(),
+            stakeholder.name
+        );
+        businessService.create();
+        cy.wait("@postBusinessService");
+        exists(businessService.name);
+
+        // Edit Business service's name
+        var updatedBusinessServiceName = data.getCompanyName();
+        businessService.edit({ name: updatedBusinessServiceName });
+        cy.wait("@getBusinessService");
+        exists(updatedBusinessServiceName);
+
+        // Delete Business service
+        businessService.delete();
+        cy.wait("@getBusinessService");
+
+        // Assert that Business service is deleted
+        notExists(businessService.name);
+
+        // Delete stakeholder owner
+        stakeholder.delete();
+        cy.wait("@getStakeholders");
+
+        // Assert that stakeholder owner is deleted
+        notExists(stakeholder.name);
```

Comment:
we can use stakeholder.email here, as name can be duplicated. So, it will cause trouble if by any chance two stakeholders with same name exists.